openscm/parameter_views.py

```diff
@@ -111,6 +112,29 @@ def get_series(self) -> Sequence[float]:
         """
         Get values of the full timeseries.
         """
+        if self.is_empty:
+            if not self._parameter._children:
+                raise ParameterEmptyError
+
+            def get_child_data(para):
+                # where should this go?
+                for name, cp in para._children.items():
```

Comment:
```suggestion
                for _, cp in para._children.items():
```
since ``name`` is unused